config/_base.js

```diff
@@ -41,6 +41,11 @@ const config = {
 // Environment
 // ------------------------------------
 // N.B.: globals added here must _also_ be added to .eslintrc
+
+const setNoMock = () => (argv.backend && config.env === 'development') || argv.noMock
```

Comment:
we have separate files for deveploment env configs. Pelase avoid such distinction in this file.